packages/bitcore-node/src/types/namespaces/ChainStateProvider.ts

```diff
@@ -27,6 +27,9 @@ export declare namespace CSP {
   export type GetBlockParams = ChainNetwork & {
     blockId: string;
   };
+  export type GetEstimateSmartFeeParams = ChainNetwork & {
```

Comment:
Lets move this down below getBlocks, I think the typings may be declared in order

Reply:
resolved